modules/oe_translation_poetry/tests/Kernel/ContactFieldsTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_translation_poetry\Kernel;
+
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\Tests\oe_translation\Kernel\TranslationKernelTestBase;
+
+/**
+ * Tests the Poetry contact fields type and formatter.
+ */
+class ContactFieldsTest extends TranslationKernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_translation_poetry_html_formatter',
+    'oe_translation_poetry',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installSchema('node', ['node_access']);
+
+    $this->container->get('entity_type.manager')->getStorage('node_type')->create([
+      'type' => 'page',
+      'name' => 'Page',
+    ])->save();
+
+    FieldStorageConfig::create([
+      'field_name' => 'organisational_contact_info',
+      'entity_type' => 'node',
+      'type' => 'oe_translation_poetry_organisation_contact',
+    ])->save();
+
+    FieldConfig::create([
+      'field_name' => 'organisational_contact_info',
+      'entity_type' => 'node',
+      'bundle' => 'page',
+    ])->save();
+
+    FieldStorageConfig::create([
```

Comment:
If each of the methods tests one of the fields, should we create both as part of the setup or rather create each field in each method?

Reply:
Ah yes indeed, let's do that.

Reply:
We merge the tests so irrelevant